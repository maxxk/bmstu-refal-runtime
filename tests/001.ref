$ENTRY Go
{
	= <Prout<Func Q>>
}


Func
{
	s.1 s.1 = 'Bad!!!';
	= 'Bad!!!';
 	e.1 = Success
}
