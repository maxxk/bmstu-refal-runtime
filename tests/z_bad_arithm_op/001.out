Operands must be same type!
